Yoast/yoast-components
# Upgrade to react 16.

## Summary

This PR can be summarized in the following changelog entry:

* Upgrade to react 16 and associated packages.

## Relevant technical choices:

* Added `enzyme-adapter-react-16` package to support react 16.

## Test instructions

This PR can be tested by following these steps:

* Run tests
* Go through all components to see nothing breaks.

Fixes #
